grpc-api/src/main/java/io/deephaven/grpc_api/session/SessionState.java

```diff
@@ -372,8 +423,8 @@ public static boolean isExportStateFinal(final ExportNotification.State state) {
         // used to manage liveness of dependencies (to prevent a dependency from being released before it is used)
         private List<ExportObject<?>> parents = Collections.emptyList();
 
-        // used to detect when this object is ready for export
-        volatile int dependentCount = -1;
+        // used to detect when this object is ready for export (is visible for atomic int field updater)
```

Comment:
Because GitHub sucks, I can't comment on code you didn't change. But `next` and `prev` should come out. Or maybe something got lost... we had a whole plan in place to get rid of the linked list running through the exported objects.